src/datreant/core/tests/test_treants.py

```diff
@@ -26,6 +27,15 @@ def treant(self, tmpdir):
             c = dtr.treants.Treant(TestTreant.treantname)
         return c
 
+    @pytest.fixture
+    def basic_treant(self, tmpdir, treantclass):
+        # treant with tags and cats, in tmpdir
+        with tmpdir.as_cwd():
+            t1 = treantclass('Rincewind')
+            t1.tags.add('magical')
+            t1.categories.add({'colour': 'octarine'})
+            yield t1
```

Comment:
Yeah that doesn't really work. You need to return a value. Which I think is ok here since pytest it doesn't delete the temp directories once they are created.